.github/workflows/ci-daily.yml

```diff
@@ -14,6 +14,7 @@ permissions: read-all
 
 jobs:
   pytest:
+    if: github.repository == 'quantumlib/Cirq'
```

Comment:
Would it be worth making the very slight simplification of testing the repository owner only (via `github.repository_owner`), instead of the owner/repo combination? That might reduce the chance of errors when copy-pasting between Quantumlib repos.

Reply:
Good point, done in 8f4e89a1b10105c0844ad03284f7e8ca12e0233a.